fxrecorder/src/lib/perfherder.rs

```diff
@@ -0,0 +1,46 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::analysis::VisualMetrics;
+use serde_json::{json, Value};
+
+/// Generate a JSON blob containing the performance metrics for Perfherder.
+pub fn generate_perfherder_metrics(metrics: &VisualMetrics) -> Value {
+    json!({
+      "application": {
+        "name": "Firefox",
+      },
+      "framework": {
+        "name": "fxrecord",
+      },
+      "suites": [
+        {
+          "name": "firstrun",
+          "subtests": [
+            {
+              "name": "SpeedIndex",
+              "value": metrics.speed_index,
+              "unit": "ms * %",
+              "lowerIsBetter": true,
+              "shouldAlert": false,
```

Comment:
Set these `shouldAlert` settings to `true`. There's currently something in treeherder/perfherder which will prevent displaying the results in the Performance tab if this if false.